### ISSUE TITLE
Support plain Django Forms

### ISSUE THREAD

**Issue by Archmonger** [fabiocaccamo/django-colorfield#211]
**Is your feature request related to a problem? Please describe.**
Currently FormField only supports `django.forms.ModelForm`, and has no variant that works with a plain `django.forms.Form`

**Describe the solution you'd like**
Either create a variant, or have the base `ColorField` be a `Form` compatible field.

```python
from colorfield.fields import ColorField
from django import forms

class BasicForm(forms.Form):
    color = ColorField(default='#FF0000')
```

**Describe alternatives you've considered**
 N/A

**Additional context**
N/A

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/fabiocaccamo) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/fabiocaccamo/django-colorfield/issues/211">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/fabiocaccamo/django-colorfield/issues/211/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/fabiocaccamo/django-colorfield/issues/211/pledge.svg">
</picture>
</a>

**Comment by fabiocaccamo** [comment]
@Archmonger thank you for reporting this, would you like to submit a PR?

**Comment by Archmonger** [comment]
I don't believe I'll have the bandwidth to PR this for at least the next 2 years.

**Comment by fabiocaccamo** [comment]
@Archmonger available in `0.12.0` version.
